airflow/operators/__init__.py

```diff
@@ -90,6 +90,7 @@
         "HiveToSambaOperator": "airflow.providers.apache.hive.transfers.hive_to_samba.HiveToSambaOperator",
     },
     "http_operator": {
+        "HttpOperator": "airflow.providers.http.operators.http.HttpOperator",
```

Comment:
One NIT. No need to add this one here. This is only needed for backwards compatiblity for those that used "airflow.operators.SimpleHttpOperator" in the past. Since this is a new operator name, nobody uses it yet, and everyone should start using `airflow.providers.http.operators.http.SimpleHttpOperator` so adding the "deprecation warning" and dynami import here makes no sense and only confuses things.